Submitty/Submitty
# [Dependency] Bump league/commonmark from 1.6.7 to 2.4.2 in /site

### Please check if the PR fulfills these requirements:


### What is the current behavior?

We couldn't bump commonmark to 2.4.0 because a no longer maintained a package that wrapped league/commonmark into a twig extension, making it easy to render the input using a filter.

### What is the new behavior?
League/commonmark is bumped to 2.4.0

### How to test?
Go to the discussion forum and make sure the markdown is rendering, so if its wrap in code blocks then the element should be rendered as a code block.
Also, make sure it is also being rendered in the notebook gradeables
